archesproject/arches-lingo
# Add login interface #13

Closes #13 

Depends on archesproject/arches#11284 and https://github.com/archesproject/arches/pull/11331

Styling will be finalized later.

### Testing instructions
Visit the site (at root), ensure you don't have a csrf cookie (first-time visitor).
  - [ ] Login works in success path
  - [ ] Appropriate message shown in failure path
  - [ ] Logout works
  - [ ] Links to register, existing auth form work
  - [ ] Redirect from auth form doesn't go to login page
  - [ ] Refresh doesn't go to login page
  - [ ] Superuser can navigate directly to django admin
